source/manual/troubleshoot-vagrant.html.md

```diff
@@ -1,6 +1,6 @@
 ---
 owner_slack: "#2ndline"
-title: Troubleshooting Vagrant
+title: Troubleshoot Vagrant
```

Comment:
Here I was just trying to make the titles match the same "Verb a noun" pattern most of the other pages follow. Does troubleshoot not work?